### PR TITLE
Adding a few more e2e tests

### PR DESCRIPTION
**Pull request by capellini**
Adding test cases to test switching back to first page using link in second page and adding test to change locales.

### CODE REVIEW

File: example/__tests__/e2e/e2e.test.js [hunk 1]
```diff
@@ -13,9 +13,35 @@ describe('example project', () => {
...
+    await expect(page).toClick('a', { text: 'Back to home' })
+    await expect(page).toMatch('A simple example')
   })
 
+  describe('changing to "de" locale', () => {
+    beforeAll(async () => {
+      await expect(page).toClick('button', { text: 'Change locale' })
+    })
+
+    it(`should display h1 in ${testLanguage} locale`, async () => {
```
Comment (isaachinman) [inline example/__tests__/e2e/e2e.test.js]: Shouldn't this be _not_ the test language?

Comment (capellini) [reply example/__tests__/e2e/e2e.test.js]: Yes - apologies.  I just fixed that.

File: example/__tests__/e2e/e2e.test.js [hunk 2]
```diff
@@ -13,9 +13,35 @@ describe('example project', () => {
...
-  it('should navigate to the second page', async () => {
+  it('should navigate to the second page and back', async () => {
     await expect(page).toClick('a', { text: 'To second page' })
     await expect(page).toMatch('A second page')
+
+    await expect(page).toClick('a', { text: 'Back to home' })
+    await expect(page).toMatch('A simple example')
   })
 
+  describe('changing to "de" locale', () => {
```
Comment (isaachinman) [inline example/__tests__/e2e/e2e.test.js]: Perhaps can we also extract `de` as a var? Maybe `testLanguageSecondary`, and change the original one to `testLanguageDefault`?

**Comment by isaachinman** [comment]
Just one nit-picky thing again... Happy to take care of it myself quickly if you don't have time. Thanks again for your help!

**Comment by capellini** [comment]
> Just one nit-picky thing again... Happy to take care of it myself quickly if you don't have time. Thanks again for your help!

No problem at all.  Requested change made and you're most welcome.

**Pull request merged**